components/Alert.css

```diff
@@ -0,0 +1,69 @@
+/* colors variables */
+
+:root {
+    --primary: rgb(0, 0, 255);
+    --primary-hover: rgb(0, 33, 140);
+    --secondary: rgb(128, 128, 128);
+    --secondary-hover: rgb(115, 115, 115);
+    --success: rgb(0, 143, 10);
+    --success-hover: rgb(0, 87, 0);
+    --danger: rgb(255, 0, 0);
+    --danger-hover: rgb(186, 0, 0);
+    --warning: rgb(255, 174, 0);
+    --warning-hover: rgb(133, 90, 0);
+    --info: rgb(0, 157, 168);
+    --info-hover: rgb(0, 90, 97);
+    --light: rgb(245, 245, 245);
+    --light-hover: rgb(230, 230, 230);
+    --dark: rgb(84, 84, 84);
+    --dark-hover: rgb(26, 26, 26);
+}
+
+
+/* *** */
+
+
+/* alert */
+
+.alert-box {
+    margin: 8px;
+    padding: 15px;
```

Comment:
try using rem , rest of the code looks fine can be merge

Reply:
sure